docs/src/aria-snapshots.md

```diff
@@ -298,18 +298,15 @@ accessible names to aid snapshot creation and review.
 
 ### 2. Updating snapshots with `@playwright/test` and the `--update-snapshots` flag
 
-When using the Playwright test runner (`@playwright/test`), you can automatically update snapshots by running tests with
-the `--update-snapshots` flag:
+When using the Playwright test runner (`@playwright/test`), you can automatically update snapshots with the `--update-snapshots` flag, `-u` for short.
+
+Running tests with the `--update-snapshots` flag will update snapshots that did not match. Matching snapshots will not be updated. Possible values are "all", "changed", "missing" and "none".
```

Comment:
I don't think the last sentence brings any clarity. Values for what? The example does not show any values. I'd drop it.